src/Microsoft.Extensions.Configuration.UserSecrets/PathHelper.cs

```diff
@@ -18,17 +18,27 @@ public class PathHelper
     {
         internal const string Secrets_File_Name = "secrets.json";
 
-#if NET451 || NETSTANDARD1_5
         /// <summary>
-        /// Gets the path to the secrets file. Uses the <paramref name="provider" /> and <seealso cref="Assembly.GetEntryAssembly" /> to find the user secrets id.
+        /// Gets the path to the secrets file. Uses the <paramref name="provider" /> and the entry assembly to find the user secrets id.
         /// </summary>
+        /// <exception cref="System.PlatformNotSupportedException">
+        /// Platforms that do not support System.Reflection.Assembly.GetEntryAssembly()
+        /// </exception>
         /// <param name="provider">The file provider</param>
         /// <returns>The filepath to secrets file</returns>
+#if NETSTANDARD1_3
+        [Obsolete("This method will always throw on this platform. Use PathHelper.GetSecretsPath(IFileProvider provider, string identifierFileName) method instead.")]
```

Comment:
Can this use a message more similar to what we have in the guidelines? https://github.com/aspnet/Home/wiki/Engineering-guidelines#breaking-changes

E.g.: `This method is obsolete and will be removed in a future version. This method will always throw an exception on this platform. The recommended alternative is PathHelper.GetSecretsPath(IFileProvider, string).`